### DISCUSSION TITLE
ESP01 controlling 2 relays,.. or not

### DISCUSSION THREAD

**Discussion by diyhouse** [arendst/Tasmota#19869] [Q&A] [answered]
I have been using the following tutorial stuff to configure an ESP01 with latest tasmota.
using;_
Program Version | 13.2.0(tasmota)
2023-10-19T09:02:07
I include a picture of the board I am using,.. which I recently purchased from well know Chinese site (Aliexpress). It has a manufacturing date of Mar 20th this year.
I have tried ( or at least I think I have ) many ( 100+ ) combinations of configs etc,.. and setting GPIO0 and 2, as the driving outputs,.. plus GPIO1 and 3 for good measure,.. as well as configuring the various rules and enabling the rule,.. but all the combinations I have tried have so far not hit the mark,.. fundamentally I am generally only getting one, relay to control,.. on the web front page interface, I can get two controls but these option still do not deliver, 
As far as I can see the i/f from the esp01 to the relays is just a pair of transistor npn driver transistors,.. so nothing too exotic.
I am aware that the ESP01 needs pins at certain levels in order to boot,.. but I am running short of ideas on where to turn next.

Does anyone have insight as to what I may be doing wrong..

![20231029_172857](https://github.com/arendst/Tasmota/assets/8927898/5b05949d-8b0b-45e6-afea-2082e9674204)

**Comment by sfromis** [comment]
The documentation for that board should tell you what pins are connected to the relays. There are 4 possible pins, gpio 0,1,2,3, and since you have one relay working, you have three possibilities for where to config relay number 2. Remember to use the dropdown number to the right to have one as number 1 and the other as number 2.

**Reply by diyhouse** [reply, marked as answer]
> Remember to use the dropdown number to the right to have one as number 1 and the other as number 2.

That's the little nugget I missed/ didn't understand.. I 'assumed' in was for configs where multiple relays were connected to a common GPIO pin, not a relay designation, as you might say.

Thanks for you feedback comments
